pkg/cmd/codespace/edit.go

```diff
@@ -44,21 +45,17 @@ func newEditCmd(app *App) *cobra.Command {
 
 // Edits a codespace
 func (a *App) Edit(ctx context.Context, opts editOptions) error {
-	codespaceName := opts.codespaceName
-
-	if codespaceName == "" {
-		selectedCodespace, err := chooseCodespace(ctx, a.apiClient)
-		if err != nil {
-			if err == errNoCodespaces {
-				return err
-			}
-			return fmt.Errorf("error choosing codespace: %w", err)
+	codespaceName, err := opts.selector.SelectName(ctx)
+	if err != nil {
+		// TODO: is there a cleaner way to do this?
+		if err == errNoCodespaces || err == errNoFilteredCodespaces {
+			return err
```

Comment:
Is there a reason why we can't also wrap those errors in 
```go
return fmt.Errorf("error choosing codespace: %w", err)
```

Reply:
That message is already on the error returned from the selector, in the case that the user actually chose something c